app/services/audit/checksum.rb

```diff
@@ -34,6 +34,8 @@ def self.validate_list_of_druids(druid_list_file_path)
 
     # validate objects with a particular status on a particular moab_storage_root
     def self.validate_status_root(status, storage_root_name)
+      raise ArgumentError, "invalid status #{status}" unless CompleteMoab.statuses.key?(status)
```

Comment:
Is this part of the Rails 7 upgrade or extra goodness?

Reply:
Hmmm. I thought I put a comment about this.

Rails used to validate the values of enums during a query. It no longer does.

There was a test for this, so above change replicates the enum check. I doubt it is crucial, but there was a test.

Reply:
🚀